### ISSUE TITLE
NWJS app v0.32.0 crashes with "nwjc -> binary.bin" usage

### ISSUE THREAD

**Issue by cryomi** [nwjs/nw.js#6742]
Hi there,

Where I've tested:
nwjs v0.32.0 sdk macos
nwjs v0.32.0 sdk linux-x64

Regularly I am using index.html as:
`
node_modules/nwjs-builder-phoenix/caches/*sdk*osx*extracted/*/nwjc dist/js.min.js dist/js.bin
`
```
<!DOCTYPE html>
<html lang="en">
    <head>
        ....
    </head>
    <body ondragstart="return false;" ondrop="return false;">
    </body>
    <script>
        nw.Window.get().evalNWBin(null, 'dist/js.bin');
    </script>
</html>
```
and it worked until v0.32.0  (v0.31.5 also is working)

But now it crashes right after start without any visible errors.
I did a test and used index.html as:
```
<!DOCTYPE html>
<html lang="en">
    <head>
        ....
        <script src="dist/js.min.js"></script>
    </head>
    <body ondragstart="return false;" ondrop="return false;">
    </body>
</html>
```

and it works fine.

**Comment by SuddenZZH** [comment]
I had the same problem

**Comment by marcgfx** [comment]
Same issue for all systems. I originally assumed I was doing something wrong, or there had been an update that requires a change to be made. I never researched the problem and just went back to 31.5 with no issue.

**Comment by Mxkkxa** [comment]
Same, I had to go back as well, can't wait to see this fixed soon!

**Comment by GnorTech** [comment]
Please provide a sample to reproduce. It works for me.

**Comment by SuddenZZH** [comment]
`loadingLayer.id = "loadingLayer";`

However, it works fine by removing this line...

**Comment by acemod13** [comment]
I checked version 0.32.1 and this version also throws the error. As soon as it starts to load the first js files (in my case, pixi.js) nwjs crashes and prints the error in the debug log.

**Comment by manhnt9** [comment]
Seems like the same issue of #6754 (have files to reproduce)

**Comment by cTn-dev** [comment]
I can confirm the whole v0.32.x series crashes when using a compiled binary inside the project.
I can also see there is quite a few issue tickets open at this time which basically point to this issue.
Is there any hotfix release planned to take care of this problem @rogerwang ?

v0.32.4 nightly still crashes

**Comment by rogerwang** [comment]
Our test cases on this passed. Will see this with any available reproducible sample.

**Comment by Jivec** [comment]
The process :
One more problem i will write it here because i think it belongs in the same category.
I have a complex scenario that i create multiple Workers in a loop, 4 at a time for optimization purposes  inside i call importNWBin in order to inject the binary code in worker context inside the compiled code i use child_process in order to spawn another 4 to do some OCR finally from there the final postmessage every time each finish. 

The problem is:
 When the first 4 workers are terminated in the next 4 workers the importNWBin is not injecting code properly. I think it has something to do with context because i'm working with a mixed-context don't know yet for sure.

Also the inline blob Worker technique doesn't work either when the code is compiled

The Question :
Did this problem/situation occurred to anyone yet.
Did anyone else had any problems with importNWBin so far or in the latest versions.

**Comment by cTn-dev** [comment]
@rogerwang from the currently open tickets i haven't really seen one with a "proper" example.
I currently compile pretty big chunks of code so finding what exactly breaks it could be a bit tricky
As of right now the v0.31.5 works fine and v0.32.0 crashes with exactly the same code.
The debug log isn't very helpful in this regard with
`[0818/025953.901:ERROR:http_transport_win.cc(175)] WinHttpCrackUrl: The URL does not use a recognized protocol (0x2ee6)`

Is there any noteworthy change that you know of between v0.31.5 and v0.32.0 that could help narrow the search down?

**Comment by Jivec** [comment]
@cTn-dev "proper" example?

**Comment by rogerwang** [comment]
merged with #6768 until a difference case provided. Please see my update on workaround there.
